### PR TITLE
DCS-806 Update docker images, dependencies and fix security audit 

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -127,7 +127,7 @@ module.exports = function createApp({
   // Secure code best practice - see:
   // 1. https://expressjs.com/en/advanced/best-practice-security.html,
   // 2. https://www.npmjs.com/package/helmet
-  app.use(helmet())
+  app.use(helmet({ contentSecurityPolicy: false }))
```
Comment (GurnankCheema) [inline server/app.js:130]: https://www.npmjs.com/package/helmet suggests the {contentSecurityPolicy : false} is already captured by helmet()

Comment (andrewrlee) [reply server/app.js:130]: The default is true, the doc tells you how to disable it which is what we're doing here

**Review by GurnankCheema — APPROVED** [review]
LGTM

**Review by bell-pa — APPROVED** [review]
LGTM

**Pull request merged**